### PR TITLE
Added moonbase alpha to list of testnets

### LINKED ISSUE #3969: Add Moonbase alphanet to the list of test networks in the front end
Name: Moonbase Alphanet
url: wss://ws.testnet.moonbeam.network
pic: tbd

### PR DESCRIPTION
**Pull request by joelamouche**
with logo and url 
closes #3969

### CODE REVIEW

**Comment by joelamouche** [comment]
@jacogr it's ready for review @crystalin

**Comment by jacogr** [comment]
We would need to add the specific types in here as well - alternatively the node would appear but the connection would fail due to mismatched types.

**Comment by joelamouche** [comment]
Thanks. Like this?

**Review by jacogr — COMMENTED** [review]
Thank you. Seems great. Just will check again on desktop before approving.

**Comment by crystalin** [comment]
Ideally, we would want Moonbase Alphanet to be available for our release of H160 AccountId, which is planned for mid November.

File: packages/apps-config/src/settings/endpoints.ts [hunk 1]
```diff
@@ -302,6 +302,12 @@ function createTestNetworks (t: TFunction): LinkOption[] {
       textBy: t('rpc.hosted.by', 'hosted by {{host}}', { ns: 'apps-config', replace: { host: 'KILT Protocol' } }),
       value: 'wss://full-nodes.kilt.io:9944/'
     },
+    {
+      info: 'moonbaseAlpha',
+      text: t('rpc.moonbeam', 'Moonbase Alpha', { ns: 'apps-config' }),
+      textBy: t('rpc.hosted.by', 'hosted by {{host}}', { ns: 'apps-config', replace: { host: 'Moonbeam Protocol' } }),
+      value: 'wss://testnet.moonbeam.network'
```
[the remark this answers is not in the document]
Comment (joelamouche) [reply packages/apps-config/src/settings/endpoints.ts]: updated

**Comment by jacogr** [comment]
@crystalin Happy to merge this at any point when you are ok, so if today or in a week from now (after the other PR), all ok. As it stands here, looks good to me.

**Comment by crystalin** [comment]
Thank you @jacogr ,
Usually how long does it take once merge to be deployed to the polkadot.js.org ?

**Comment by jacogr** [comment]
10-15 mins, depending on CI. All merges are deployed immediately on success. (IPFS is updated weekly, generally on Monday since that is when "stable" releases are made)

**Comment by joelamouche** [comment]
@jacogr please merge this whenever you can if you are still satisfied with this PR

**Comment by jacogr** [comment]
Thank you. Should be there in 10-15 mins after this - https://github.com/polkadot-js/apps/actions/runs/358081298

**Pull request merged**